docs/EN_US/ContainerizedHPCC/ContainerizedMods/CustomConfig.xml

```diff
@@ -882,18 +885,26 @@ thor: []
           <listitem>
             <para>Storage Cost</para>
           </listitem>
+         <listitem>
+            <para>Compile Cost</para>
+          </listitem>
 
           <listitem>
             <para>File Access Cost</para>
           </listitem>
+          
+          <listitem>
+            <para>File Cost at Rest</para>
```

Comment:
Storage Cost is listed as a separate item in the list of costs above.  Storage cost isn't a separate cost item. Storage Cost = File Access Cost + File Cost At Rest.  Also, the Execution cost is not listed.  The Types of costs tracked are:
- Execute Cost
- Compile Cost
- File Access Cost
- File Cost At Rest